tmt/steps/provision/__init__.py

```diff
@@ -1365,6 +1373,23 @@ def is_ready(self) -> bool:
         # Enough for now, ssh connection can be created later
         return self.primary_address is not None
 
+    def setup(self) -> None:
+        if self.is_dry_run:
+            return
+        if not self.facts.is_superuser and self.become:
+            assert self.facts.package_manager is not None
+            self.execute(
```

Comment:
Please, add a TODO comment pointing to https://github.com/teemtee/tmt/pull/2557. Yet another place to switch to a true package manager.